about.html

```diff
@@ -0,0 +1,119 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About :: First Capstone Project</title>
+    <!-- fontawesome css1 -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="./assets/images/icon.svg" rel="icon" type="img/svg" sizes="16x16"/>
+    <link rel="stylesheet" href="assets/css/about.css">
+</head>
+<body>
+    <header class="header">
+        <section class="top-nav">
+            <ul class="row container">
+                <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class="fa-brands fab fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </section>
+        <nav class="navbar">
+            <div class="mobile-overlay"></div>
+            <div class="mobile-foreground"></div>
+            <section class="container nav">
+                <div class="logo">
+                    <a href="./"><img src="assets/images/icon.svg" alt="logo" class="logo"></a>
+                </div>
+                <ul class="bars">
+                    <li class="bar-item"></li>
+                    <li class="bar-item"></li>
+                    <li class="bar-item"></li>
+                </ul>
+                <ul class="nav-link">
+                    <li class="home-link"><a href="./">Home</a></li>
+                    <li><a class="active" href="./about.html">About</a></li>
+                    <li><a href="#program">Program</a></li>
+                    <li><a href="#speaker">Speakers</a></li>
+                    <li><a href="#contact">Sponsor</a></li>
+                    <li><a href="#contact">News</a></li>
+                    <li><a href="#" class="btn btn-outline-primary">Campaign</a></li>
+                </ul>
+            </section>
+        </nav>
+        <section class="hero container text-center">
+            <p class="greeting">"Hello! Sharing world"</p>
+            <h1>MV FULL-STACK <br> GRADUATION EVENT 2023 </h1>
+            <div class="description">
+                <p>The Global Event brings together the community of professional, and activists who comprise the Spiritual World affiliate network in a different region every year. </p><p class="desc-desktop">This year, we hope to expand our invitation list including organizations and individuals. who want to participate and shared their experiences.</p><p class="desc-desktop">Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi libero at optio placeat qui quia veniam eius deleniti. Impedit vitae incidunt delectus aspernatur tempore pariatur expedita minus sunt natus quo.</p>
```

Comment:
- [ ] I would like to recommend you change the background color to the color defined in the original design. It's important to stick the original design as much as possible  :+1:

Reply:
Thank you @codecaiine , i will make all the changes and request a new review